include/boost/url/experimental/format.hpp

```diff
@@ -0,0 +1,147 @@
+//
+// Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/url
+//
+
+#ifndef BOOST_URL_EXPERIMENTAL_FORMAT_HPP
+#define BOOST_URL_EXPERIMENTAL_FORMAT_HPP
```

Comment:
This should not be in "experimental" since it is not an experimental feature. And all new public includes need to be added to `<boost/url.hpp>`